src/Redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import dragonSlice from './dragons/dragonSlice';
+
+const rootReducer = combineReducers({
+  dragons: dragonSlice,
+});
+
+const store = configureStore({
+  reducer: rootReducer,
+});
+
+export default store;
```

Comment:
could you please  move the entire code to the feature instead of Redux since all the team agreed to work on this file